todo_app/templates/index.html

```diff
@@ -23,6 +23,17 @@ <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items %}
           <li>{{ item.title }} - {{ item.status}}</li>
+          <form action="/update_status" method="Post" class="form-example">
+            <div class="form-example">
+              <p hidden>
+                <lable for="name"> Todo Id: </lable>
+                <input type="text" name="todo-id" value="{{ item.id }}" required>
+              </p>
+            </div>
+            <div class="form-example">
+              <input type="submit" class="btn btn-success" value="Complete">
+            </div>
```

Comment:
This can be simplified
- No reason to have a hidden label. (also typo label -> lable).
- No reason for the `p` or the `div`s
- No reason for `required` on an input when you are providing the value yourself anyway

You can instead hide the item id input with `type="hidden"`:
```suggestion
              <input type="hidden" name="todo-id" value="{{ item.id }}">
              <input type="submit" class="btn btn-success" value="Complete">
```